### PR TITLE
new Screen for Categories

### PR DESCRIPTION
**Pull request by rafaelgtraverso**
**DO NOT MERGE**
@jgtoriginal For some reason is not working on ios, and check ANDROID as well because it has some Ref error ( hope you can reproduce it)

### CODE REVIEW

File: App.js [hunk 1]
```diff
@@ -17,6 +17,7 @@ import { setNavigator } from './src/navigationRef';
 import LoadingScreen from './src/screens/LoadingScreen';
 import configureStore from './src/store';
 import PasswordRecoveryScreen from './src/screens/PasswordRecoveryScreen';
+import CardsScreen from './src/screens/CardsScreen';
```
Comment (jgtoriginal) [inline App.js:20]: this file doesn't exist @rafaelgtraverso

Comment (jgtoriginal) [reply App.js:20]: image

Comment (rafaelgtraverso) [reply App.js:20]: Sorry for some reason it wasn't include in the push.

**Comment by jgtoriginal** [comment]
@rafaelgtraverso it works for me.

However I don't see a category filter or navigator. So once I click on a card of the first set, it takes me to the second set and doesn't provide me a way to go back to the first one.

phrases

File: src/screens/CardsScreen.js [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+      />
+    </View>
+  );
+};
+
+CardsScreen.propTypes = {
+  show_phrase: PropTypes.func,
+  phrases: PropTypes.object,
+  cat_name: PropTypes.string,
+  cat_cards: PropTypes.array,
```
Comment (jgtoriginal) [inline src/screens/CardsScreen.js:37]: `cat_cards` is declared but not used
however `cards_cat` you got further up, which is not declared here.

File: src/components/CardsGrid.js [hunk 3]
```diff
@@ -29,7 +26,9 @@ const CardsGrid = props => {
         data={data}
         renderItem={({ item }) => (
           <TouchableOpacity onPress={() => on_Press(item)}>
-            <Card item={item}/>
+            <Card
```
Comment (jgtoriginal) [inline src/components/CardsGrid.js:29]: no need to indent here.

**Review by jgtoriginal — APPROVED** [review]

**Pull request merged**